modules/nw-metallb-configuring-frr-8ks.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-frr-k8s.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-metallb-configuring-frr-k8s_{context}"]
+= Activating the integration of MetalLB and FRR-K8s
+
+The following procedure shows you how to activate `FRR-K8s` as the backend for `MetalLB`.
+
+.Prerequisites
+
+* You have a cluster installed on bare-metal hardware.
+* You have installed the OpenShift CLI (`oc`).
+* You have logged in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+* Set the `bgpBackend` field of the `MetalLB` CR to `frr-k8s` as in the following example:
```

Comment:
This will be default in 4.17. Maybe you can mention it?

Reply:
we shouldn't and we should not make any promises, we never  know :-)

Reply:
yes with docs we're generally advised against promising things in future releases

Reply:
got it!